internal/cmd/run/espresso.go

```diff
@@ -140,6 +142,21 @@ func applyEspressoFlags(p *espresso.Project, flags espressoFlags) error {
 	}
 
 	if p.Suite.Name == "" {
+		isErr := len(p.Suite.TestOptions.Class) != 0 ||
+			len(p.Suite.TestOptions.NotClass) != 0 ||
+			p.Suite.TestOptions.Package != "" ||
+			p.Suite.TestOptions.Size != "" ||
+			p.Suite.TestOptions.Annotation != "" ||
+			p.Suite.TestOptions.NotAnnotation != "" ||
+			p.Suite.TestOptions.NumShards != 0 ||
+			p.Suite.TestOptions.UseTestOrchestrator ||
+			flags.Device.Changed ||
+			flags.Emulator.Changed
```

Comment:
How about introducing some validators package to do the validation? like https://github.com/go-playground/validator

Reply:
I'll check it out, thanks! 👍